### PR TITLE
Optionally load ResourceOwner from OIDC user info endpoint

### PR DESCRIPTION
**Pull request by pradtke**

### CODE REVIEW

**Comment by shadowhand** [comment]
Would this fix #34?

**Comment by pradtke** [comment]
It took a while for me to find time to test this. Yes, this change should fix that issue. You don't seem to need Google Plus access to use the OIDC endpoints.

In my testing I can disable Google Plus access for an api key/secret and still load name, email, sub, image from the OIDC user info endpoint (https://www.googleapis.com/oauth2/v3/userinfo), while if I try with the Google Plus endpoint I get that error.

File: test/src/Provider/GoogleTest.php [hunk 1]
```diff
@@ -20,6 +30,15 @@ protected function setUp()
             'hostedDomain' => 'mock_domain',
             'accessType' => 'mock_access_type'
         ]);
+
+        $this->providerOidc = new GoogleProvider([
```
Comment (shadowhand) [inline test/src/Provider/GoogleTest.php]: Instead of adding this to the existing test, can you make a new one for the different configuration?

Comment (pradtke) [reply test/src/Provider/GoogleTest.php]: I've created a separate test class.

**Comment by shadowhand** [comment]
Thanks!

**Comment by shadowhand** [comment]
Released in 2.1.0.

**Pull request merged**